### PR TITLE
Adding classname prop to ObjectDetail

### LINKED ISSUE #29023: Object Detail View is no longer scrollable inside Query Builder
A regression since https://github.com/metabase/metabase/pull/28728 is that the Object Detail view no longer scrolls if we have more content than can be shown on the screen. 

**To Reproduce**
Steps to reproduce the behavior (if you can reproduce the bug using the Sample Database, we will find the issue faster):
1. Go to New -> Question -> Orders -> Join Products and People
2. Visualize and select the Detail view. Notice That there is no scroll bar and you cannot reach the bottom of the list

**Expected behavior**
The visualization should be scrollable

image

*Metabase Installation*
image

### PR DESCRIPTION
**Pull request by npfitz**
Closes https://github.com/metabase/metabase/issues/29023

### Description
The `Visualization.jsx` adds a `className` to card it renders. Passing this to the Object Detail resolved the issue of no scrollbars being applied when the container was larger than the screen.

### How to verify
Repro details are in the linked issue

### Demo
image

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: e2e/test/scenarios/visualizations/object_detail.cy.spec.js [hunk 1]
```diff
@@ -178,6 +181,49 @@ describe("scenarios > question > object details", () => {
...
+          ],
+        },
+        type: "query",
+      },
+    };
+    visitQuestionAdhoc(questionDetails);
+
+    cy.findByTestId("object-detail");
+
+    cy.log("metabase(#29023)");
```
Comment (EmmadUsmani) [inline e2e/test/scenarios/visualizations/object_detail.cy.spec.js:223]: Is this log meant to be here permanently?

**Review by EmmadUsmani — APPROVED** [review]

File: e2e/test/scenarios/visualizations/object_detail.cy.spec.js (hunk 1 above)
Comment (npfitz) [reply e2e/test/scenarios/visualizations/object_detail.cy.spec.js:223]: Yeah, I just wanted a way to show that that checking the scrolling was related to that issue, but dedicating the entire test to it seemed a bit much

Comment (EmmadUsmani) [reply e2e/test/scenarios/visualizations/object_detail.cy.spec.js:223]: Sounds good 👍

**Review by iethree — APPROVED** [review]
Good solution! 🥇

**Pull request merged**